src/flatpickr.js

```diff
@@ -722,6 +722,25 @@ function Flatpickr(element, config) {
 	function onKeyDown(e) {
 		if (self.isOpen) {
 			switch (e.which) {
+				case 9: 
+					let days = document.querySelector("div.flatpickr-days");
```

Comment:
The days reference will be likely incorrect in case there are multiple calendars.
Use self.days instead